### PR TITLE
zh-trans:/content/zh/about/contribute/add-content/index.md

### PR DESCRIPTION
**Pull request by gauliang**
ref:https://github.com/servicemesher/istio-official-translation/issues/1766

- [x] Update to 1.5

### CODE REVIEW

File: content/zh/about/contribute/add-content/index.md [hunk 1]
```diff
@@ -10,170 +10,132 @@ aliases:
...
+- 读者在执行任务之前必须理解一个概念：在执行任务之前，请向他们介绍先决条件信息并提供指向该信息的链接。
 
-- The reader needs to make a decision: Provide the conceptual information
-  necessary to know when to make the decision, the available options, and when
-  to choose one option instead of the other.
+- 读者需要做出决定：提供必要的概念性信息，以了解何时做出决定，可用选项以及何时选择一个选项而不是另一个。
 
-- The reader is an administrator but not a SWE: Provide a script,
-  not a link to a code sample in a developer’s guide.
+- 读者是管理员，而不是SWE：提供执行脚本，而不是开发人员指南中的代码示例链接。
```
Comment (gorda) [inline content/zh/about/contribute/add-content/index.md]:
```suggestion
- 读者是运维人员，而不是软件工程师：提供执行脚本，而不是开发人员指南中的代码示例链接。
```

SWE，我猜测是 SoftWare Engineer 的缩写

File: content/zh/about/contribute/add-content/index.md [hunk 2]
```diff
@@ -10,170 +10,132 @@ aliases:
...
-        <li>Posts highlighting Istio features.</li>
-        <li>Posts detailing how to accomplish a task or fulfill a specific use case using Istio. Unlike Tasks and Examples, the technical accuracy of blog posts is not maintained and tested after publication.</li>
+        <li>文章详细介绍了作者使用和配置 Istio 的经验，尤其是那些表达新颖经验或观点的经验。</li>
+        <li>突出 Istio 功能的文章。</li>
+        <li>文章详细介绍了如何使用 Istio 完成任务或实现特定用例。与任务和示例不同，博客文章的技术准确性在发布后不会得到维护和测试。</li>
         </ul>
       </td>
-      <td>Readers with a basic understanding of the project who want to learn
-      about it in an anecdotal, experiential, and more informal way.</td>
+      <td>对项目有基本了解的读者想通过案例、体验和更加非正式的方式来了解它。</td>
```
Comment (gorda) [inline content/zh/about/contribute/add-content/index.md]:
```suggestion
      <td>对项目有基本了解的读者想通过案例、体验和更加无拘束的方式来了解它。</td>
```

**Comment by gauliang** [comment]
@gorda 可以 Approve 了

**Comment by gauliang** [comment]
@googlebot I sign it!

**Pull request merged**